clients/traycer-cli/src/service/platforms/windows.ts

```
@@ -195,7 +203,33 @@ async function installService(
+      // observed it - deliberately even when it replaces a `/Create` failure,
+      // since "may not mutate at all" is the more fundamental fact of the
+      // two. After `/Create` succeeded it is post-registration, and every
+      // post-registration throw is marked (`didServiceRegistrationCommit`),
+      // by reference so the error keeps its identity.
+      if (!isServiceMutationAuthorityError(cause)) {
+        createCliLogger(options.label.environment).debug(
+          "Failed to remove the staged task definition; leaving it behind",
+          { task: taskName, cause: describeCause(cause) },
+        );
+      } else {
+        throw created ? markRegistrationCommitted(cause) : cause;
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)